src/Core/Models/Business/OrganizationLicense.cs

```diff
@@ -150,7 +157,14 @@ public OrganizationLicense(Organization org, SubscriptionInfo subscriptionInfo,
     /// Represents the current version of the license format. Should be updated whenever new fields are added.
     /// </summary>
     /// <remarks>Intentionally set one version behind to allow self hosted users some time to update before
-    /// getting out of date license errors</remarks>
+    /// getting out of date license errors
+    /// </remarks>
+    /// <remarks>
+    /// ⚠ This is important! We are on this license version because of new
+    /// properties that were added for versions 14 and 15. These properties
+    /// were later removed! When you increment to version 16 please delete
+    /// this comment.
+    /// </remarks>
```

Comment:
I don't understand the purpose of this comment. Maybe it no longer applies given that we haven't actually removed the properties. (only marked them as deprecated)

Reply:
Yeah, this should be removed. Done!

__Commit:__ [Remove remark about license](https://github.com/bitwarden/server/pull/4730/commits/768a362b15c181adeeefef220a40038772650a60)